### PR TITLE
Update Zopier SIP domain configuration

### PR DESCRIPTION
**Pull request by vikalejo**
- SIP domain needs to be registered
- Username to authenticate could be a name, username or phone number

### CODE REVIEW

File: guides/tools/soft-phone.md [hunk 1]
```diff
@@ -13,13 +13,18 @@ Before you configure the Zoiper client you have to create SIP domain on [Twilio
 - SIP URI - URI which you will need to use for Zoiper configuration later
 - REQUEST URL - URL to your application which will handle API requests (see [how to configure ngrok](https://github.com/DVELP/cookbook/blob/master/guides/tools/ngrok.md) to get public url from your localhost)
 
+You need to register the SIP domain.
```
Comment (vitaliiprodan) [inline guides/tools/soft-phone.md:16]: Is it required step?

File: guides/tools/soft-phone.md [hunk 2]
```diff
@@ -13,13 +13,18 @@ Before you configure the Zoiper client you have to create SIP domain on [Twilio
...
+ For instance for North America Virginia (US1) will be `{domain-name}.sip.us1.twilio.com`.
+
 ### Authentication
 
 Soft-phones require basic authentication, so we have to create a user and password. On Twilio:
 - Click on `+` near with CREDENTIAL LISTS field
 - Fill in the form:
   - FRIENDLY NAME - preferably your name
-  - USERNAME - enter your phone number in E164 format e.g. +441231231231
+  - USERNAME - Username to authenticate (name, username, phone_number),
```
Comment (vitaliiprodan) [inline guides/tools/soft-phone.md:27]: I would prefer phone number. In this case we will be able to extract phone number using this method https://github.com/DVELP/twilio_base/blob/master/lib/twilio_base/helpers/protocols.rb#L8

File: guides/tools/soft-phone.md (hunk 1 above)
Comment (vikalejo) [reply guides/tools/soft-phone.md:16]: It was when I tested it. Zopier recognized the Sip when registered. 
But after has the Zopier with the account registered, disabling the Registration looks to work fine so it is not a required step.

File: guides/tools/soft-phone.md (hunk 2 above)
Comment (vikalejo) [reply guides/tools/soft-phone.md:27]: got it , didnt know about that, I thought it was an isolated utility
Let's keep the phone number then.
Thanks

**Comment by vikalejo** [comment]
No needed changes:
- SIP registration is not a required step. 
- We use the phone number value as a username as we manipulate that attribute in twilio_base.

**Pull request closed without merging**